test/helpers/random.js

```diff
@@ -0,0 +1,7 @@
+const { ethers } = require('hardhat');
+
+function randomHex(length) {
+  return `0x${Buffer.from(ethers.randomBytes(length)).toString('hex')}`;
+}
```

Comment:
This can be replaced with 
```
ethers.hexlify(ethers.randomBytes(length))
``` 
I would remove that file and use the line above directly